app-tasks/rf/src/rf/uploads/geotiff/create_thumbnails.py

```diff
@@ -0,0 +1,121 @@
+"""Creates thumbnails for geotiff imagery
+
+Depends on having geotiff file already downloaded
+"""
+import os
+import subprocess
+
+import boto3
+
+from rf.models import Thumbnail
+
+from .io import get_geotiff_dimensions
+
+def create_thumbnails(tif_path, scene_id, organization_id):
+    """Creates thumbnails based on a geotiff path and scene
+
+    Args:
+        tif_path (str): path of local tiff file
+        scene_id (str): uuid of the scene
+
+    Returns:
+        List[Thumbnail]
+    """
+
+    r_tif_path = 'resampled_%s' % tif_path
+    rp_tif_path = 'reprojected_%s' % tif_path
+
+    dim = get_geotiff_dimensions(tif_path)
+    max_dim = float(max(get_geotiff_dimensions(tif_path)))
+
+    size_large = 1024
+    size_small = 256
+
+    scale_large = size_large / max_dim
+    scale_small = size_small / max_dim
+
+    dim_large = (dim[0] * scale_large, dim[1] * scale_large)
+    dim_small = (dim[0] * scale_small, dim[1] * scale_small)
+
+    # Create paths for each size thumb
+    path_large = '%s-LARGE.png' % scene_id
+    path_small = '%s-SMALL.png' % scene_id
+
+    # Create urls for each size Thumbnail
+    url_large = '/thumbnails/%s' % path_large
+    url_small = '/thumbnails/%s' % path_small
+
+    try_to_remove([r_tif_path, rp_tif_path, path_large, path_small])
+
+    if os.path.isfile(tif_path):
+        # Resample tif
+        subprocess.call([
```

Comment:
Should these be `subprocess.check_call` so that this bombs out if any of the `gdal*` calls fail?

Side-note: It looks like `rasterio` supports warping and resampling via the GDAL API, so I think it would be possible to do this in Python, which I would normally recommend over using `subprocess`. However, looking at the `rasterio` documentation, I think for this use case that calling `subprocess` is actually going to be clearer and more succinct, so I'm :+1: for this.